src/handlers/inGameCommandHandler.js

```diff
@@ -168,13 +168,15 @@ module.exports = {
                         rustplus.printCommandOutput(`Turning ${groupName} off.`);
```

Comment:
These off/on to OFF/ON as well, squash the commits into one and it should be good to go.

Reply:
You can squash when you merge, just select "[Squash & Merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request)".